services/ansibler/Dockerfile

```diff
@@ -36,6 +36,10 @@ RUN CGO_ENABLED=0 go build
 
 #Use alpine based ansible image
 FROM cytopia/ansible:2.13
+#Add repository label
+LABEL org.opencontainers.image.source https://github.com/berops/claudie
```

Comment:
I like the OCI labels.
however, I have only ever used them in the following format:
```
LABEL org.opencontainers.image.created=$BUILD_DATE \
      org.opencontainers.image.licenses=MIT
       ...etc...
```
that is assigning the values to the keys (please disregard the fact that they are grouped under a single LABEL call).

can we verify (`docker inspect`) that the labels are present in the images?

Reply:
```
"Labels": {
                "org.opencontainers.image.description": "Image for Context-box from Claudie",
                "org.opencontainers.image.source": "https://github.com/berops/claudie"
            }
```

Reply:
I have troubles building Builder's the image locally, the Dockerfile seems to be tightly coupled with the CI logic.

Reply:
> I have troubles building Builder's the image locally, the Dockerfile seems to be tightly coupled with the CI logic.

Try `docker build --tag ghcr.io/berops/claudie/builder:wULL -f ./services/builder/Dockerfile .` from root of the repo

Reply:
thanks @MiroslavRepka, I figured that was the issue, I guess I'm too used to building Dockerfiles from the current folder... :see_no_evil: 

![image](https://user-images.githubusercontent.com/61180606/211606018-2d1c269b-1624-4146-97fe-616361b42f29.png)

btw could we perhaps add the `org.opencontainers.image.base.name` label as well, while we're at it?